backend/src/fun_forecast_backend/activity_schema/dataclasses.py

```diff
@@ -1,23 +1,39 @@
 from dataclasses import dataclass
-from typing import List
+from typing import List, Dict
 
 
 @dataclass
-class Range:
-    low: int
-    high: int
-    with_weather: List[int]
+class Wind:
+    min: int
+    max: int
+    ideal: int
+    units: str
+
+
+@dataclass
+class Temp:
```

Comment:
nit: Wind and Temp are currently equivalent, feels like they could be combined. Then again, not sure what I would call that dataclass 🤔